prefect_dbt/cloud/clients.py

```diff
@@ -0,0 +1,72 @@
+"""Module containing clients for interacting with the dbt Cloud API"""
+from typing import Optional
+
+import prefect
+from httpx import AsyncClient, Response
+
+from prefect_dbt.cloud.models import TriggerJobRunOptions
+
+
+class DbtCloudAdministrativeClient:
```

Comment:
Can you explain the purpose of this client class?

Is this class's purpose only to implement async context manager?  If so, I think this works:
```
from dataclasses import dataclass
from httpx import AsyncClient

import prefect


@dataclass
class DbtCloudCredentials():
    
    api_key: str
    account_id: int
    domain: str = "cloud.getdbt.com"

    def get_client(self):
        return AsyncClient(
            headers={
                "Authorization": f"Bearer {self.api_key}",
                "user-agent": f"prefect-{prefect.__version__}",
            },
            base_url=f"https://{self.domain}/api/v2/accounts/{self.account_id}",
        )


credentials = DbtCloudCredentials("api_key", "account_id")

async with credentials.get_client() as client:
    print(client)
```